mail-filter/rspamd/files/rspamd-3.0-system-doctest.patch

```diff
@@ -0,0 +1,72 @@
+From d37a3764e9889e6de0a7341ba3195ee8a314de23 Mon Sep 17 00:00:00 2001
```

Comment:
I've said it before but I'd like to just say again that you're doing great work, and it is really appreciated. Thanks.

Reply:
Thanks :)